x-pack/plugins/secops/server/utils/build_query/merge_fields_with_hits.ts

```diff
@@ -6,10 +6,10 @@
 
 import { get, has, merge } from 'lodash/fp';
 
-export const mergeFieldsWithHit = (
+export const mergeFieldsWithHit = <T>(
```

Comment:
👍 Thanks for the fix. Sorry for changing the generated types when I shouldn't have.